vehicle/mkvi/software/charging/spi_api.c

```diff
@@ -0,0 +1,174 @@
+// look up MCP25625 datasheet to see how to deal with SPI and message transmission
```

Comment:
oops I didn't realize how much work y'all put into making the api interface - prob should delete this tho since we're using the external library :(